elements/rh-card/rh-card.css

```diff
@@ -51,37 +87,15 @@
   box-sizing: border-box;
 }
 
-:host([alt]) #container.dark {
+:host([color-palette~="dark"]) #container.dark {
   --_background-color: var(--rh-color-surface-dark, #383838);
 }
 
-:host([alt]) {
+:host([color-palette~="light"]) {
   --_background-color: var(--rh-color-surface-light, #e0e0e0);
   --_header-color: var(--rh-color-text-primary-on-light, #151515);
 }
 
-:host([bar]) #header {
```

Comment:
Good point.  I guess when we're considering patterns versus variants it would come down to is it functionally doing anything (in this case, no) but the styles are preset in a way that does alter the header element pretty significantly.  Do we have a wiki page or anything (if not, I can create one) about when we'd make something a variant vs a pattern?  Can also bring it up in standup

Reply:
we don't have clear guidance on this yet, i must admit

absent clear guidance, I'd say "less is more" - we can always add later, but removing APIs is very difficult to justify